packages/letters/letters.0.1.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Client library for sending email over SMTP"
+description: "Simple to use SMTP client implementation for ocaml"
+maintainer: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+authors: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+homepage: "https://github.com/oxidizing/letters/"
+bug-reports: "https://github.com/oxidizing/letters/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.0"}
+  "mrmime" {= "0.3.0"}
+  "colombe" {= "0.3.0"}
+  "sendmail-lwt" {= "0.3.0"}
+  "fmt" {= "0.8.8"}
```

Comment:
These constraints seem rather strict. Your project seems rather useful, but potential users might get a bit annoyed if they can't have their up-to-date library and install your project. Opam only allows one version to be installable in a switch.

Could you consider changing all the `=` constraints to a `>=`? Our CI will check your package on future releases of the dependencies. We modify the constraints accordingly it is effectively broken with a new release.

Reply:
I was playing a bit and testing how dependency constraints work. I'll remove all others except `mrmime`, `colombe` and `sendmail-lwt`. APIs of those three aren't very stable state so fixing versions like this is pretty much needed.

Reply:
Fixed with the exceptions I commented above.